packages/console/src/icons/CheckBoxSelected.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { SVGProps } from 'react';
+
+const CheckBoxSelected = (props: SVGProps<SVGSVGElement>) => (
+  <svg
+    width="20"
+    height="20"
+    viewBox="0 0 20 20"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+    {...props}
+  >
+    <rect x="1.66602" y="1.66663" width="16.6667" height="16.6667" rx="4" fill="#5D34F2" />
+    <path
+      fillRule="evenodd"
+      clipRule="evenodd"
+      d="M8.31476 13.858L5.13295 10.441C4.95568 10.253 4.95568 9.947 5.13295 9.757L5.77568 9.074C5.95295 8.886 6.24113 8.886 6.4184 9.074L8.63657 11.466L13.5811 6.141C13.7584 5.953 14.0465 5.953 14.2238 6.141L14.8665 6.825C15.0438 7.013 15.0438 7.32 14.8665 7.507L8.95748 13.858C8.78021 14.046 8.49203 14.046 8.31476 13.858Z"
+      fill="white"
+    />
+  </svg>
+);
```

Comment:
You don't have to create tsx icons any more. We are now using @parcel/tramsformer-svg, all the `.svg` file can be directly imported and used as a react component

Reply:
They are just aligned with other icons.
Maybe add an issue to refactor them all?

Reply:
But make sure we don't add more .tsx from now on

Reply:
Yes, no more `SVG's in the remaining iteration tasks.